ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -60,11 +61,20 @@ $services.localization.render('contrib.meeting.home.presentation', ['[[**', '**&
   'translationPrefix': 'meeting.livetable.',
   'tagCloud': true,
   'rowCount': 15,
-  'maxPages': 10,
-  'selectedColumn': 'startDate',
-  'defaultOrder': 'desc'
+  'maxPages': 10
```

Comment:
In the https://extensions.xwiki.org/xwiki/bin/view/Extension/Livetable+Macro#HParameter24options, you will find that the default for `maxPages` is `10`, so you can remove it from here, as it is redundant.